geps/gep-1619/index.md

```diff
@@ -59,6 +59,68 @@ session affinity, as this design is expected to be addressed within a separate G
 
 ## Introduction
 
+### Naming
+
+Naming is hard. We've had lots of [discussion](https://github.com/kubernetes-sigs/gateway-api/discussions/2893) on the
+topic of naming session persistence. Adding to the complexity, the Gateway API implementations do not have a consensus
+on a naming convention for session persistence or affinity.
+
+To start this discussion, lets establish the idea of strong session affinity (what this GEP calls session persistence)
+and weak session affinity (what this GEP calls session affinity) which we will define further in [The Relationship of Session Persistence and Session Affinity](#the-relationship-of-session-persistence-and-session-affinity).
+In this context, "strong" implies a guarantee, while "weak" indicates a best-effort approach.
+
+Here's a survey of how some implementations refers to these ideas:
```

Comment:
Nit: May want to include Envoy itself here, which uses "Stateful Session" terminology https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_filters/stateful_session_filter

Reply:
Good point. But they also refer to it as "session stickiness". I'll put both.